backend/webserver/urls.py

```diff
@@ -36,4 +38,6 @@
     path('authors/<str:pk>/posts/<int:post_id>/',PostView.as_view()),
     path('authors/<str:pk>/posts/',AllPosts.as_view()),
     path('posts/', AllPublicPostsView.as_view()),
+    path('authors/<str:pk>/posts/<int:post_id>/likes/',LikePostView.as_view()),
+    path('authors/<str:pk>/liked/',LikedPostView.as_view()),
```

Comment:
```suggestion
    path('authors/<str:pk>/liked/', AuthorLikedView.as_view()),
```
This view should not be named `LikedPostView` because an author can like both comments and posts and the same view would process both kinds of likes.